domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityComplexFieldsValidatorImplTest.java

```diff
@@ -20,13 +21,14 @@
 import java.util.Arrays;
 
 import static junit.framework.TestCase.assertFalse;
```

Comment:
This is the JUnit4 version of assertFalse.  Change it to `import static org.junit.Assert.assertFalse;`